logging_demo/package.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>logging_demo</name>
+  <version>0.0.3</version>
+  <description>Examples for composing multiple nodes in a single process.</description>
```

Comment:
The description should be updated, maintainer tag as well

Reply:
do you think it's alright to start this at v0.0.3? figured it's easier for when we bump them all?

Reply:
yup! will make @nuclearsandwich's blooming easier

Reply:
All packages in the same repository need to have the same version, so not keeping it in line with the other package's version would actually make it impossible to release :)

Reply:
ah! ok then it stays!